modules/lotamePanoramaIdSystem.js

```diff
@@ -14,10 +14,41 @@ const KEY_EXPIRY = `${KEY_ID}_expiry`;
 const KEY_PROFILE = '_cc_id';
 const MODULE_NAME = 'lotamePanoramaId';
 const NINE_MONTHS_MS = 23328000 * 1000;
-const DAYS_TO_CACHE = 7;
-const DAY_MS = 60 * 60 * 24 * 1000;
 
 export const storage = getStorageManager(null, MODULE_NAME);
+const cookieDomain = findCookieDomain(String(document.domain));
```

Comment:
Any tests that need to be added to the codebase?

Reply:
Yes, I found a way and added some